tests/test_saltworker.py

```diff
@@ -405,7 +409,18 @@ def test_windows_install(saltworker_base_salt_args):
     assert saltworker_win.cleanup.call_count == 1
 
 
-@patch.dict(os.environ, {"systemdrive": "C:"})
+@patch.dict(os.environ, {"systemdrive": "C:",
+                         "ProgramFiles": "C:\\Program Files"})
+@patch("os.path.isfile", MagicMock(return_value=True))
+def test_windows_install_new():
```

Comment:
Let's name this test something closer to what it is testing:

```suggestion
def test_windows_salt_call_new():
```